### ISSUE TITLE
Mermaid (pre-release) - Vertical flowchart cut off in viewer

### ISSUE THREAD

**Issue by dpoulton-github** [laurent22/joplin#2637]
- Joplin 1.0.187 (Portable) Pre-Release
- Windows 10 Pro x64 (Virtual Machine)

As discussed on the Joplin forum.

## Steps to Reproduce

Create a series of vertical flowcharts using Mermaid (See markdown below). Ensure one does not have a "node" or a "text label" in the left arm. If a vertical flowchart does not have a “node” or “label” on a left-hand flow, the left of the chart disappears off the left of the viewer.

![mermaid-flow](https://user-images.githubusercontent.com/60117449/75695986-c652e480-5ca2-11ea-9153-5b3a21f0abe8.png)

## Expected Behaviour

The flowchart should position itself to be completely visible.

### Markdown Used To Test

```text
graph TD
     a --> b;
     a --> c;
     b --> d;
     c --> d;

graph TD
     a --> c;
     a --> d;
     c --> d;

graph TD
     a --> c;
     c --> d;
     a --> d;
```

**Comment by laurent22** [comment]
This appears to be a bug in Mermaid - the line is not just off screen, it's simply not drawn at all. Perhaps something to report to them.

**Comment by tessus** [comment]
FYI: Not fixed in Mermaid 8.4.8.

**Comment by tessus** [comment]
see mermaid-js/mermaid/issues/1293
